cve_bin_tool/OutputEngine.py

```diff
@@ -62,6 +64,56 @@ def output_csv(self, outfile):
                     row = [modulename, version, cve_number, cve_severity]
                     writer.writerow(row)
 
+    def write_console(self, modulename, version, cve_number, cve_severity, outfile):
+        """ returns console version"""
+        space1 = " " * (15 - len(modulename))
```

Comment:
These numbers (`15` and friends) I believe are connected to the number of `-` in end_string. Could you make that more clear by assigning them to variables such as `MODULE_NAME_MAX = 15`.

Also, what happens if the `modulename` (or any other variable) is longer than it's max? You probably want to use [`textwrap.shorten`](https://docs.python.org/3.7/library/textwrap.html#textwrap.shorten)

Reply:
No, they are not connected to the ```-```  but they do ensure that all the variables are fit according to the max_tabular_size. Considering the problem of the variable name being longer than max_size I have tried text wrap but I couldn't figure out how to fit that into my logic. Although I have set if the size is larger let the table to disfigure but show all the variables as it is Eg would look like 
```
+--------------------------+----------------------+------------+------------+
| A very long  Module name fits | version is also long | CVE-XXXX-XXXXXXXXXX | MEDIUM |
+-------------------------------+----------------------+---------------------+--------+
```
end_string consisting of + and - is generated according to the length of variables